webpack.config.js

```diff
@@ -166,6 +169,11 @@ module.exports = function (env = {}) {
     },
     optimization: {
       minimize: env.production,
+      minimizer: [
+        new EsbuildPlugin({
+          target: buildTarget, // Syntax to transpile to (see options below for possible values)
```

Comment:
```suggestion
          target: buildTarget,
```

Reply:
did you sanity check the size of the js modules in the devtools (master vs this branch)?